packages/react/src/components/Forms/Select/tokens.ts

```diff
@@ -15,6 +15,7 @@ export const tokens = (defaults: DefaultTokens) => ({
   "item-background-color--hover": defaults.theme.colors["greyscale-200"],
   "item-background-color--selected": defaults.theme.colors["primary-100"],
   "item-color": defaults.theme.colors["greyscale-800"],
+  "item-color--disabled": defaults.theme.colors["greyscale-600"],
```

Comment:
I'll use something lighter as from my point of view contract between enable and disable options is too low.

Reply:
I think they tried to keep a sufficient contrast between text and its background. But in this case I wonder what is the proper solution ?
Have a good constrast ratio between enabled and disabled options or have a good constrast ratio between all kind of options and their background ?

Reply:
I added a fixup with `greyscale-500`, wdyt?

Reply:
👍